images/linux/scripts/base/apt.sh

```diff
@@ -18,3 +18,9 @@ apt-get install jq
 add-apt-repository -y ppa:apt-fast/stable
 apt-get update
 apt-get install apt-fast
+
+# Install retry logic for apt and apt-get
+sudo mv apt-get-retry.sh /usr/local/sbin/apt-get
```

Comment:
Why do we need to mv operation if we already copied this file?
        {
            "type": "file",
            "source": "{{template_dir}}/scripts/base/apt-get-retry.sh",
            "destination": "/usr/local/sbin/apt-get"
        },